internal/backend/httpsvc/intercept.go

```diff
@@ -51,7 +54,54 @@ func (r *responseInterceptor) Flush() {
 
 type RequestLogExtractor func(*http.Request) []zap.Field
 
-func intercept(z *zap.Logger, cfg *LoggerConfig, extractors []RequestLogExtractor, next http.Handler) (http.HandlerFunc, error) {
+var metrics = struct {
```

Comment:
did you try the other option of having just one counter like ```api_request_total_count``` with path/status dimensions ? 
and maybe another one ```api_request_duration``` with path/status dimension as well ?

Reply:
let's talk